perl/lib/Sanger/CGP/PindelPostProcessing/FragmentFilterRules.pm

```diff
@@ -500,8 +502,8 @@ sub flag_020 {
     return $PASS;
   }
 
-  my $tumfc_over_tumfd = $tum_geno[$previous_format_hash->{'FD'}] > 0 ? $tum_geno[$previous_format_hash->{'FC'}] / $tum_geno[$previous_format_hash->{'FD'}] : undef;
-  my $norfc_over_norfd = $nor_geno[$previous_format_hash->{'FD'}] > 0 ? $nor_geno[$previous_format_hash->{'FC'}] / $nor_geno[$previous_format_hash->{'FD'}] : undef;
+  my $tumfc_over_tumfd = $tum_geno[$previous_format_hash->{'FD'}] > 0 ? $tum_geno[$previous_format_hash->{'FC'}] / $tum_geno[$previous_format_hash->{'FD'}] : -1;
+  my $norfc_over_norfd = $nor_geno[$previous_format_hash->{'FD'}] > 0 ? $nor_geno[$previous_format_hash->{'FC'}] / $nor_geno[$previous_format_hash->{'FD'}] : -1;
```

Comment:
Check your logic, if `norfc_over_norfd` is <=0.05 it can be used to pass, not fail.

I'd be wary of passing any event where FD>FC

Reply:
undef is no better in this case

```
$ perl -e '$x=undef; if($x < 5){print qq{hello\n};}'
hello
```